### PR TITLE
Issue/2327 remove cruft

### PR DESCRIPTION
**Pull request by danahertzberg**
This is amazing. No HTML cruft!

### CODE REVIEW

File: modules/custom/az_marketing_cloud/templates/views-view--az-marketing-cloud.html.twig [hunk 1]
```diff
@@ -0,0 +1 @@
+{{rows}}
```
Comment (danahertzberg) [inline modules/custom/az_marketing_cloud/templates/views-view--az-marketing-cloud.html.twig:1]:
```suggestion
{{ rows }}
```

File: modules/custom/az_core/az_core.module [hunk 2]
```diff
@@ -266,7 +266,6 @@ function az_core_canonical_url(array $data) {
   $node = $data['node'];
   /** @var \Drupal\Core\Url $url */
   $url = $node->toUrl();
-  $url->setAbsolute(TRUE);
   $uri = $url->toString();
```
Comment (danahertzberg) [inline modules/custom/az_core/az_core.module:269]: Was originally `$uri = $url->toUriString();` is that a problem if we change core?

File: modules/custom/az_marketing_cloud/az_marketing_cloud.module [hunk 3]
```diff
@@ -68,6 +94,62 @@ function az_marketing_cloud_preprocess_node__view__az_marketing_cloud(&$variable
...
+    foreach ($previous as $p) {
+      $suggestions[] = $p . '__' . $view_id;
+    }
+  }
+  if($route_name === 'view.az_marketing_cloud.50_50_layout') {
+    if($variables['elements']['#id'] === 'az_barrio_content') {
+      $suggestions[] = 'block__az_marketing_cloud';
+    }
+  }
+  if($route_name === 'view.az_marketing_cloud.30_70_layout') {
```
Comment (joeparsons) [inline modules/custom/az_marketing_cloud/az_marketing_cloud.module:128]:
```suggestion
  if ($route_name === 'view.az_marketing_cloud.30_70_layout') {
```

File: modules/custom/az_marketing_cloud/az_marketing_cloud.module [hunk 4]
```diff
@@ -68,6 +94,62 @@ function az_marketing_cloud_preprocess_node__view__az_marketing_cloud(&$variable
...
+  $view_id = \Drupal::routeMatch()->getParameter('view_id');
+  $route_name = \Drupal::routeMatch()->getRouteName();
+  if (!empty($view_id)) {
+    $previous = $suggestions;
+    $view_id = strtr($view_id, '-', '_');
+    foreach ($previous as $p) {
+      $suggestions[] = $p . '__' . $view_id;
+    }
+  }
+  if($route_name === 'view.az_marketing_cloud.50_50_layout') {
```
Comment (joeparsons) [inline modules/custom/az_marketing_cloud/az_marketing_cloud.module:123]:
```suggestion
  if ($route_name === 'view.az_marketing_cloud.50_50_layout') {
```

File: modules/custom/az_marketing_cloud/az_marketing_cloud.module [hunk 5]
```diff
@@ -68,6 +94,62 @@ function az_marketing_cloud_preprocess_node__view__az_marketing_cloud(&$variable
...
+    if($variables['elements']['#id'] === 'az_barrio_content') {
```
Comment (joeparsons) [inline modules/custom/az_marketing_cloud/az_marketing_cloud.module:124]:
```suggestion
    if ($variables['elements']['#id'] === 'az_barrio_content') {
```

File: modules/custom/az_marketing_cloud/az_marketing_cloud.module [hunk 6]
```diff
@@ -68,6 +94,62 @@ function az_marketing_cloud_preprocess_node__view__az_marketing_cloud(&$variable
...
+      $suggestions[] = 'block__az_marketing_cloud';
+    }
+  }
+  if($route_name === 'view.az_marketing_cloud.30_70_layout') {
+    if($variables['elements']['#id'] === 'az_barrio_content') {
```
Comment (joeparsons) [inline modules/custom/az_marketing_cloud/az_marketing_cloud.module:129]:
```suggestion
    if ($variables['elements']['#id'] === 'az_barrio_content') {
```

File: modules/custom/az_marketing_cloud/az_marketing_cloud.module [hunk 7]
```diff
@@ -68,6 +94,62 @@ function az_marketing_cloud_preprocess_node__view__az_marketing_cloud(&$variable
...
+      $suggestions[] = 'block__az_marketing_cloud';
+    }
+  }
+  if($route_name === 'view.az_marketing_cloud.text_layout') {
```
Comment (joeparsons) [inline modules/custom/az_marketing_cloud/az_marketing_cloud.module:133]:
```suggestion
  if ($route_name === 'view.az_marketing_cloud.text_layout') {
```

File: modules/custom/az_marketing_cloud/az_marketing_cloud.module [hunk 8]
```diff
@@ -68,6 +94,62 @@ function az_marketing_cloud_preprocess_node__view__az_marketing_cloud(&$variable
...
+    if($variables['elements']['#id'] === 'az_barrio_content') {
```
Comment (joeparsons) [inline modules/custom/az_marketing_cloud/az_marketing_cloud.module:134]:
```suggestion
    if ($variables['elements']['#id'] === 'az_barrio_content') {
```

File: modules/custom/az_marketing_cloud/az_marketing_cloud.module [hunk 9]
```diff
@@ -68,6 +94,62 @@ function az_marketing_cloud_preprocess_node__view__az_marketing_cloud(&$variable
...
+      $suggestions[] = 'block__az_marketing_cloud';
+    }
+  }
+}
+
+function az_marketing_cloud_theme_suggestions_views_view_alter(array &$suggestions, array $variables) {
+  $route_name = \Drupal::routeMatch()->getRouteName();
+  if($route_name === 'view.az_marketing_cloud.50_50_layout') {
```
Comment (joeparsons) [inline modules/custom/az_marketing_cloud/az_marketing_cloud.module:142]:
```suggestion
  if ($route_name === 'view.az_marketing_cloud.50_50_layout') {
```

File: modules/custom/az_marketing_cloud/az_marketing_cloud.module [hunk 10]
```diff
@@ -68,6 +94,62 @@ function az_marketing_cloud_preprocess_node__view__az_marketing_cloud(&$variable
...
+      $suggestions[] = 'views_view__az_marketing_cloud';
+  }
+  if($route_name === 'view.az_marketing_cloud.30_70_layout') {
```
Comment (joeparsons) [inline modules/custom/az_marketing_cloud/az_marketing_cloud.module:145]:
```suggestion
  if ($route_name === 'view.az_marketing_cloud.30_70_layout') {
```

File: modules/custom/az_marketing_cloud/az_marketing_cloud.module [hunk 11]
```diff
@@ -68,6 +94,62 @@ function az_marketing_cloud_preprocess_node__view__az_marketing_cloud(&$variable
...
+    $suggestions[] = 'views_view__az_marketing_cloud';
+  }
+  if($route_name === 'view.az_marketing_cloud.text_layout') {
```
Comment (joeparsons) [inline modules/custom/az_marketing_cloud/az_marketing_cloud.module:148]:
```suggestion
  if ($route_name === 'view.az_marketing_cloud.text_layout') {
```

File: modules/custom/az_marketing_cloud/az_marketing_cloud.module [hunk 12]
```diff
@@ -68,6 +94,62 @@ function az_marketing_cloud_preprocess_node__view__az_marketing_cloud(&$variable
...
+ */
+function az_marketing_cloud_theme_suggestions_region_alter(array &$suggestions, array $variables) {
+  $route_name = \Drupal::routeMatch()->getRouteName();
+  if($route_name === 'view.az_marketing_cloud.50_50_layout') {
+    $suggestions[] = 'region__az_marketing_cloud';
+  }
+  if($route_name === 'view.az_marketing_cloud.30_70_layout') {
+    $suggestions[] = 'region__az_marketing_cloud';
+  }
+  if($route_name === 'view.az_marketing_cloud.text_layout') {
```
Comment (joeparsons) [inline modules/custom/az_marketing_cloud/az_marketing_cloud.module:108]:
```suggestion
  if ($route_name === 'view.az_marketing_cloud.text_layout') {
```

File: modules/custom/az_marketing_cloud/az_marketing_cloud.module [hunk 13]
```diff
@@ -68,6 +94,62 @@ function az_marketing_cloud_preprocess_node__view__az_marketing_cloud(&$variable
...
 
+/**
+ * Implements hook_theme_suggestions_HOOK_alter().
+ */
+function az_marketing_cloud_theme_suggestions_region_alter(array &$suggestions, array $variables) {
+  $route_name = \Drupal::routeMatch()->getRouteName();
+  if($route_name === 'view.az_marketing_cloud.50_50_layout') {
+    $suggestions[] = 'region__az_marketing_cloud';
+  }
+  if($route_name === 'view.az_marketing_cloud.30_70_layout') {
```
Comment (joeparsons) [inline modules/custom/az_marketing_cloud/az_marketing_cloud.module:105]:
```suggestion
  if ($route_name === 'view.az_marketing_cloud.30_70_layout') {
```

File: modules/custom/az_marketing_cloud/az_marketing_cloud.module [hunk 14]
```diff
@@ -68,6 +94,62 @@ function az_marketing_cloud_preprocess_node__view__az_marketing_cloud(&$variable
   }
 }
 
+/**
+ * Implements hook_theme_suggestions_HOOK_alter().
+ */
+function az_marketing_cloud_theme_suggestions_region_alter(array &$suggestions, array $variables) {
+  $route_name = \Drupal::routeMatch()->getRouteName();
+  if($route_name === 'view.az_marketing_cloud.50_50_layout') {
```
Comment (joeparsons) [inline modules/custom/az_marketing_cloud/az_marketing_cloud.module:102]:
```suggestion
  if ($route_name === 'view.az_marketing_cloud.50_50_layout') {
```

**Review by joeparsons — COMMENTED** [review]
Added a bunch of nitpicky coding standards suggestions.

**Pull request merged**